content/en/docs/concepts/storage/volumes.md

```diff
@@ -900,6 +900,21 @@ receive Secret updates.
 
 For more details, see [Configuring Secrets](/docs/concepts/configuration/secret/).
 
+### serviceAccountToken
+
+A [serviceAccount](/docs/tasks/configure-pod-container/configure-service-account/) 
+is used to provide an identity for pods. Pods that want to interact with the API 
+server will authenticate using a particular ServiceAccount. By default, applications 
+will authenticate as the `default` ServiceAccount in the namespace they are running 
+in. For instance, that an application running in the `test` namespace will use the 
+default `ServiceAccount` in the `test` namespace.
```

Comment:
there isn't a top-level serviceaccounttoken volume type... it is one of the possible sources under projected volumes, which appear to already be documented at https://kubernetes.io/docs/concepts/storage/projected-volumes/

I think it would make more sense to add documentation about the service account token volume to that page.

For the specific docs, I would drop the bit about the default service account (`By default, applications will authenticate as the `default` ServiceAccount in the namespace they are running in. For instance, that an application running in the `test` namespace will use the default `ServiceAccount` in the `test` namespace.`) and just link to the service account docs.

Reply:
This text could go into https://kubernetes.io/docs/concepts/storage/projected-volumes/ pretty much unamended.

@mehabhalodiya would you be willing to adapt this change to target the right page?

Reply:
I think the change to this file should still be dropped

Reply:
Something similar to this text should go into the projected volumes page. Right now, there are 4 items in https://kubernetes.io/docs/concepts/storage/projected-volumes/#introduction and one of them is missing a place to link to. Let's fix that.

Reply:
adding a linkable header around ~line 47 of content/en/docs/concepts/storage/projected-volumes.md and linking to that could work

Reply:
changes to content/en/docs/concepts/storage/projected-volumes.md look good. drop the change to this file since serviceAccountToken is not a volume type (it's only available in a projected volume)